packages/components/src/components/Passphrase/PassphraseTypeCardHeading.tsx

```diff
@@ -0,0 +1,107 @@
+import { ReactNode } from 'react';
+
+const IconWrapper = styled.div<{ $type: WalletType }>`
+    width: 38px;
+    height: 38px;
+    background: ${({ theme, $type }) =>
+        $type === 'standard'
+            ? theme.backgroundPrimarySubtleOnElevation1
+            : theme.backgroundSurfaceElevation1};
+    border-radius: ${borders.radii.sm};
+    display: flex;
+    justify-content: center;
+    align-items: center;
+`;
+
+const Description = styled.div<{ $hasTopMargin?: boolean }>`
+    display: flex;
+    color: ${({ theme }) => theme.textSubdued};
+    ${typography.label}
+`;
+
+const ArrowCol = styled(Rows)`
+    flex: 0 0 auto;
+    height: 100%;
+    justify-content: center;
+`;
+
+const WalletTitle = styled.div<{ $withMargin: boolean }>`
+    display: flex;
+    ${typography.body}
+    color: ${({ theme }) => theme.textDefault};
+    align-items: center;
+    ${props => props.$withMargin && `margin-bottom: ${spacingsPx.xxs};`}
+`;
+
+type PassphraseTypeCardHeadingProps = {
+    title?: ReactNode;
+    description?: ReactNode;
+    type: WalletType;
+    learnMoreTooltipOnClick?: TooltipProps['addon'];
+    learnMoreTooltipAppendTo?: TooltipProps['appendTo'];
+};
+
+export const PassphraseTypeCardHeading = ({
+    type,
+    learnMoreTooltipOnClick,
+    learnMoreTooltipAppendTo,
+    title,
+    description,
+}: PassphraseTypeCardHeadingProps) => {
+    const theme = useTheme();
+
+    return (
+        <Columns gap={spacings.xl}>
+            <IconWrapper $type={type}>
+                {type === 'standard' ? (
+                    <Icon size={24} icon="WALLET" color={theme.iconPrimaryDefault} />
+                ) : (
+                    <Icon size={24} icon="LOCK" color={theme.iconSubdued} />
+                )}
+            </IconWrapper>
+            <Rows justifyContent="center" flex="1">
+                <WalletTitle
+                    $withMargin={type === 'hidden'}
+                    data-test={type === 'hidden' && '@tooltip/passphrase-tooltip'}
+                >
+                    {type === 'hidden' ? (
+                        <Tooltip
+                            appendTo={learnMoreTooltipAppendTo}
+                            title={
+                                <FormattedMessage
+                                    id="TR_WHAT_IS_PASSPHRASE"
+                                    defaultMessage="Learn more about the difference"
+                                />
+                            }
+                            addon={learnMoreTooltipOnClick}
+                            content={
+                                <FormattedMessage
+                                    id="TR_HIDDEN_WALLET_TOOLTIP"
+                                    defaultMessage="Passphrases add a custom phrase (e.g. a word, sentence, or string of characters) to your recovery seed. This creates a hidden wallet; each hidden wallet can use its own passphrase. Your standard wallet will still be accessible without a passphrase."
```

Comment:
Just curious: do we need those `defaultMessage` in the code? We have them in `message.jsom` file so here it seems to me as unnecessary stuff that pollutes code.